itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/adapter/CDKServerAdapterAbstractTest.java

```diff
@@ -81,14 +80,23 @@
 @DisableSecureStorage
 @RemoveCDKServers
 @OpenPerspective(value=JBossPerspective.class)
+@CleanDockerExplorer
 public abstract class CDKServerAdapterAbstractTest extends CDKAbstractTest {
 
 	protected ServersView2 serversView;
 	
 	protected Server server;
 	
+	private static boolean certificateAccepted = false;
+	
 	public static final String OPENSHIFT_USERNAME = "developer"; //$NON-NLS-1$
 	
+	public static final String OPENSHIFT_ADMIN = "admin"; //$NON-NLS-1$
+	
+	public static final String MINISHIFT_PROFILE2 = "minishift2"; //$NON-NLS-1$
+	
+	public static final String SKIP_REGISTRATION = "--skip-registration"; //$NON-NLS-1$
```

Comment:
@odockal "--skip-registration" could be probably moved to CDKAbstractTest or in labels class, you are using it also in another class